lib/puma/launcher.rb

```diff
@@ -81,7 +83,7 @@ def initialize(conf, launcher_args={})
       set_rack_environment
 
       if clustered?
-        @events.formatter = Events::PidFormatter.new
+        @events.formatter = Events::PidFormatter.new unless @options[:log_formatter]
```

Comment:
Actually, let's just make this `@events.formatter ||= Events::PidFormatter.new`. I don't like have two conditionals looking at `options[:log_formatter]`.

Reply:
Oh right, duh, there's already a formatter.

Instead, let's just do it all in one conditional then. Keep our impact in as much of one place as possible.

```ruby
@events.formatter = if options[:log_formatter]
  options[:log_formatter]
elsif clustered?
  Events::PidFormatter
end
```

Reply:
Did something a little different so I don't have to add an else case with the default formatter